firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/common/IdManagerTest.java

```diff
@@ -75,6 +77,23 @@ public void testCreateUUID() {
     assertEquals(installId, idManager.getCrashlyticsInstallId());
   }
 
+  public void testGetIdExceptionalCase() {
+    FirebaseInstallationsApi iid = mock(FirebaseInstallationsApi.class);
```

Comment:
Nit: can we call this `fis` instead of `iid`?

Reply:
done